docs/plantuml/CalendarShowCommandSequence.puml

```diff
@@ -0,0 +1,65 @@
+@startuml CalendarShowCommandSequence
```

Comment:
I think you should put this inside `diagrams` instead, else asciidoctor will try to process this file as well.